### PR TITLE
Update README.md 

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR updates the README.md file and removes the clusterConfig references from the Liqo documentation.

# How Has This Been Tested?

Not applicable.

### CODE REVIEW

File: docs/pages/gettingstarted/extended/install.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ liqoctl commands take a couple of minutes to complete. If liqoctl returns succes
 
 Liqo should be installed on all three clusters:
```
Comment (giorio94) [inline docs/pages/gettingstarted/extended/install.md:66]:
```suggestion
Liqo should be installed on all three clusters.
```

File: README.md [hunk 2]
```diff
@@ -70,6 +70,23 @@ Liqo is a platform to enable dynamic and decentralized resource sharing across K
...
+```
+
+Then, you should execute the following commands to install the latest version of liqoctl:
+
+```
+curl --fail -LSO "https://get.liqo.io/liqoctl-${OS}-${ARCH}" && \
+chmod +x "liqoctl-${OS}-${ARCH}" && \
+sudo mv "liqoctl-${OS}-${ARCH}" /usr/local/bin/liqoctl
+```
+
```
Comment (giorio94) [inline README.md:89]: You might add the same sentence of the doc, about downloading from the release page

File: docs/pages/configuration/configure.md [hunk 3]
```diff
@@ -5,28 +5,14 @@ weight: 1
...
-To modify the `ClusterConfig` via kubectl use the following command:
-```bash
-kubectl edit clusterconfig
-```
-and modify the field: 
-```yaml
-discoveryConfig: 
-   clusterName: your_cluster_name
-```
+You can easily modify it whenever you want using `liqoctl install`. The `liqoctl install` workflow is idempotent and can be executed multiple obtain to enforce the desired configuration.
```
Comment (giorio94) [inline docs/pages/configuration/configure.md:10]:
```suggestion
You can easily modify it whenever you want using `liqoctl install`. The `liqoctl install` workflow is idempotent and can be executed multiple times to enforce the desired configuration.
```

File: docs/pages/configuration/configure.md [hunk 4]
```diff
@@ -5,28 +5,14 @@ weight: 1
...
 
 ```
-# Create a new namespace named 'liqo-demo'
-kubectl create namespace liqo-demo
-# Associate the 'liqo.io/enabled' label to the above namespace
-kubectl label namespace liqo-demo liqo.io/enabled=true
+liqoctl install ${YOUR_PROVIDER} --cluster-name ${YOUR_CLUSTER_NAME}
 ```
+
+where `${YOUR_PROVIDER}` is the provider for your cluster and `${YOUR_CLUSTER_NAME}` is the name you want to assign
```
Comment (giorio94) [inline docs/pages/configuration/configure.md:18]:
```suggestion
where `${YOUR_PROVIDER}` is the provider for your cluster and `${YOUR_CLUSTER_NAME}` is the name you want to assign.
```

File: docs/pages/configuration/discovery.md [hunk 5]
```diff
@@ -24,24 +24,20 @@ Besides, mDNS discovery implies also that your cluster is discoverable by others
 
 #### Enable and Disable the discovery on LAN
 
-The discovery on LAN can be enabled and disabled by updating the flags in the ClusterConfig CR. Lan Discovery can be 
-disabled to avoid unwanted peering with neighbors.
+Lan Discovery is enabled by default. You can disable it by setting to false the `enable-lan-discovery` to liqoctl install:
```
Comment (giorio94) [inline docs/pages/configuration/discovery.md:27]:
```suggestion
LAN discovery is enabled by default. You can disable it by setting to false the `enable-lan-discovery` flag of liqoctl install.
```

File: docs/pages/configuration/discovery.md [hunk 6]
```diff
@@ -24,24 +24,20 @@ Besides, mDNS discovery implies also that your cluster is discoverable by others
...
-The automatic LAN discovery is enabled by default.
+where `${YOUR_PROVIDER}` is the provider for your cluster and `${YOUR_CLUSTER_NAME}` is the name you want to assign to it.
+
+If you want to turn it on again you just have to:
+
+```
+liqoctl install ${YOUR_PROVIDER} --cluster-name ${YOUR_CLUSTER_NAME} --enable-lan-discovery=true
+```
 
 ### Manual Configuration
```
Comment (giorio94) [inline docs/pages/configuration/discovery.md:42]: This part should be also updated to use `liqoctl`

**Review by giorio94 — APPROVED** [review]

File: docs/pages/configuration/discovery.md (hunk 6 above)
Comment (palexster) [reply docs/pages/configuration/discovery.md:42]: :+1: Will be part of the follow-up PRs on documentation

**Review by aleoli — APPROVED** [review]

**Pull request merged**